aws/tagsGreengrass.go

```diff
@@ -0,0 +1,125 @@
+package aws
```

Comment:
Drive-by comment: Hi @oleksiyVeretiuk 👋 FYI, the recommended methodology for working with resource tagging is the new `keyvaluetags` package, which offers a consistent interface to handle all this (generally boilerplate) code for you and allows the maintainers to more easily perform changes across the entire provider -- you can find more information about this in the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#adding-resource-tagging-support). See also: https://github.com/terraform-providers/terraform-provider-aws/issues/10688

Please note that if the `keyvaluetags` package additions are submitted in a separate pull request, we generally merge those in pretty quickly and these resource pull requests can be rebased with that functionality available.